pytest_instafail.py

```diff
@@ -46,7 +46,8 @@ def pytest_collectreport(self, report):
         # Show errors occurred during the collection instantly.
         TerminalReporter.pytest_collectreport(self, report)
         if report.failed:
-            self.rewrite("")  # erase the "collecting" message
+            if self.isatty:
```

Comment:
Or just `if getattr(self, "isatty", True):` - though do you really think it makes sense to still keep 2.8 compatibility?